qa/TL0_python_self_test_frameworks/test_cupy.sh

```diff
@@ -5,6 +5,7 @@ target_dir=./dali/test/python
 
 test_body() {
     nosetests --verbose -m '(?:^|[\b_\./-])[Tt]est.*cupy' test_dltensor_operator.py
+    nosetests --verbose test_gpu_python_function_operator.py
```

Comment:
Note: If you name your file as `test_operator_*.py` it is picked automatically by oython self test (not sure if that is what you want)

Reply:
I don't want to, because i cannot import cupy in python self test